functests/pvc_creation_test.go

```diff
@@ -15,7 +15,7 @@ import (
 
 var _ = Describe("PVC Creation", func() {
 	var k8sClient *kubernetes.Clientset
-
+	var deployManager *deploymanager.DeployManager
```

Comment:
Add a newline after this. Also make the same change in `data_validation_test.go`.

Reply:
They are being **declared**  
https://github.com/openshift/ocs-operator/blob/48295f5bbaa5fa89fceb0c44f4e32ef5bbe1f4ec/functests/data_validation_test.go#L22
and on Line 51
and they are being used in their own way in both of those functions
So I believe, we don't need a declaration there

Reply:
It can be set on line 22, that's what we want, but the variable needs to be *declared* at the top-level `Describe()`, just after the `var k8sClient` declaration.